examples/detection/yolov2/forward.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
ChainerCV follows coding convention of Chainer.
Could you please remove `coding: utf-8 `?